src/Client/Updater.php

```diff
@@ -575,4 +525,70 @@ private function getUpdateStartTime(): \DateTimeImmutable
     {
         return (new \DateTimeImmutable())->setTimestamp($this->clock->getCurrentTime());
     }
+
+    /**
+     * Performs a delegation search for a target metadata object.
+     *
+     * @param \Tuf\Metadata\TargetsMetadata|null $targetsMetadata
+     *   The targets metadata to search.
+     * @param string $target
+     *   The path of the target file. Needs to be known to the most recent
+     *   targets metadata downloaded in ::refresh().
+     * @param string[] $searchedRoles
+     *   The roles that have already been searched. This is for internal use only and should not be passed by calling code.
+     *   calls to this function and should be provided by any callers.
+     * @param bool $terminated
+     *
+     * @return \Tuf\Metadata\TargetsMetadata|null
+     *   The target metadata that is the end node for search or null if the target metadata does not make any delegations.
```

Comment:
I'd rephrase this too. Maybe "The targets metadata that was the last visited node in the delegation tree, or null if there were no delegated roles to search. If targets metadata is returned, it may or may not contain information about the desired target."

Reply:
I simplified this method so only returns a targets metadata object if it has the info about the file. So the comment is simpler now